### ISSUE TITLE
Account Dropdown Menu Not Working in Firefox

### ISSUE THREAD

**Issue by delta-plus** [CiviWiki/OpenCiviWiki#172]
After logging into the site on Firefox, the account dropdown menu in the top right corner doesn't display when moused over or clicked. This makes logging out or accessing account settings difficult unless you type the path directly into the browser. Tested with the latest version of Firefox on both Linux and Windows.

**Comment by jl24** [comment]
I'm unable to replicate this bug on Firefox Quantum 59.0.2 (64-bit) on OSX and on Windows. Tested the current dev branch. @delta-plus was that the branch you were on?

**Comment by delta-plus** [comment]
I was just using the regular website. I'll try to replicate it on some
computers at my college today.

**Comment by delta-plus** [comment]
Okay, just tried it on a computer at my college campus. Chrome is fine,
Firefox still won't do the dropdown. In the JS web console, an error is
logged upon clicking the dropdown button: "ReferenceError: event is not
defined".

**Comment by jl24** [comment]
Ok I can see the bug in action now in production.

**Comment by jl24** [comment]
Looks like it was solved in a previous commit (b08075d). And that will be pushed to production once milestone 0.2 is complete. This unfortunately means that navigation on Firefox will be disabled until then 😞. We can probably discuss more about our methods of prioritizing/deploying hotfixes immediately to production, but for now, since the problem had been resolved previously I'm going to close the issue.

**Comment by delta-plus** [comment]
Okay, cool. Nice job on the quick resolution!
